Product/Production/Services/DocumentSubmissionCore/src/main/java/gov/hhs/fha/nhinc/docsubmission/inbound/deferred/request/PassthroughInboundDocSubmissionDeferredRequest.java

```diff
@@ -80,7 +81,7 @@ XDRAcknowledgementType processDocSubmissionRequest(ProvideAndRegisterDocumentSet
             dsUtils.convertDataToFileLocationIfEnabled(body);
             response = sendToAdapter(body, assertion);
         } catch (LargePayloadException lpe) {
-            LOG.error("Failed to retrieve payload document.", lpe);
+            LOG.error("Failed to retrieve payload document." + lpe.getLocalizedMessage(), lpe);
```

Comment:
Minor readability issue: can you make sure there's a space before concatenating the exception message? (check other locations as well)